src/Reflectivity/ProcessorScheduler.extension.st

```diff
@@ -0,0 +1,15 @@
+Extension { #name : 'ProcessorScheduler' }
+
+{ #category : '*Reflectivity' }
+ProcessorScheduler >> allProcesses [
+
+	^ Process allSubInstances
+]
+
+{ #category : '*Reflectivity' }
+ProcessorScheduler >> findContextsForMethod: aMethod [
+	"Take care: users should call within #valueUnpreemptively"
+
+	^ self allProcesses flatCollect: [ :process |
+		  process findContextsForMethod: aMethod ]
+]
```

Comment:
This method is actually not used so there will be no incidence. It could be deleted.

I tried in a test to collect all processes that were suspended, but it was useless because I use semaphores in my tests and it didn't collect a process suspended by a semaphore